src/uni_modules/wot-ui/components/wd-input/index.scss

```
@@ -1,35 +1,35 @@
 @use '../../styles/mixin/mixin.scss'as *;
 @use '../../styles/variable.scss'as *;
 // 输入框内边距
-$input-padding: var(--wot-input-padding, $padding-loose $padding-loose);
+$input-padding: var(--wot-input-padding, $padding-loose $padding-loose) !default;
 // 输入框背景色
```

Stylelint (17.10.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)